book/team.yaml

```diff
@@ -203,4 +203,27 @@ people:
         link: https://github.com/jomey
     user_groups:
       - Technology Specialist
-  # =============
\ No newline at end of file
+  # =============
+   - title: Romina Piunno
+    avatar: https://avatars.githubusercontent.com/u/17185856?v=4
+    role: Graduate Student
+    organizations:
```

Comment:
thanks @RomiP - column level of indentation needs to match indentation of above entries